### ISSUE TITLE
Typo in AWSCognitoAuth.m assigning signInRedirectUri

### ISSUE THREAD

**Issue by devTechi** [aws-amplify/aws-sdk-ios#1090]
Hello,

There is a typo in creation of the AWSCognitoAuth.m file. It prevents the user from connecting to Cognito because of a wrong SingIn-Redirecting url.

In order to work as expected please change the [line 1005](https://github.com/aws-amplify/aws-sdk-ios/blob/master/AWSCognitoAuth/AWSCognitoAuth.m#L1005) from

```ObjC
    return [self initWithAppClientId:appClientId appClientSecret:appClientSecret scopes:scopes signInRedirectUri:signOutRedirectUri signOutRedirectUri:signOutRedirectUri webDomain:webDomain identityProvider:identityProvider idpIdentifier:idpIdentifier userPoolIdForEnablingASF:userPoolIdForEnablingASF enableSFAuthSessionIfAvailable:NO];
```

to

```ObjC
    return [self initWithAppClientId:appClientId appClientSecret:appClientSecret scopes:scopes signInRedirectUri:signInRedirectUri signOutRedirectUri:signOutRedirectUri webDomain:webDomain identityProvider:identityProvider idpIdentifier:idpIdentifier userPoolIdForEnablingASF:userPoolIdForEnablingASF enableSFAuthSessionIfAvailable:NO];
```

Please note the change from `signInRedirectUri:signOutRedirectUri` to `signInRedirectUri:signInRedirectUri`.

Thanks

**Comment by palpatim** [comment]
@devTechi Thanks very much for finding this, and for so quickly identifying the fix!

I also appreciate that you took it upon yourself to fix as part of PR #1091. Since that PR is quite a bit larger than just this fix, and since this fix represents a regression, we're going to fix the regression on a smaller, targeted commit so we can give your PR more devoted attention without the time pressure of this regression hanging over us.

**Comment by devTechi** [comment]
Hi @palpatim,
you're welcome.

Will you make the changes by yourself or shall I create another isolated PR?

**Comment by palpatim** [comment]
Sorry for the confusion, we have made the changes and are preparing a release.

**Comment by palpatim** [comment]
This is fixed in 2.6.35
